vscode-extension/package.json

```diff
@@ -119,34 +119,42 @@
           {
             "id": "initializeExtension",
             "title": "Initialize Extension",
-            "description": "Get the extension set up for first use.\n[Initialize Extension](command:vscode-aiconfig.init)",
+            "description": "Ensure you have everything needed before using the extension for the first time\n[Initialize Extension](command:vscode-aiconfig.init)",
             "media": {
               "markdown": "media/initializeExtension.md"
-            }
+            },
+            "completionEvents": ["onCommand:vscode-aiconfig.init"]
           },
           {
             "id": "setEnvironment",
             "title": "Set API Keys",
-            "description": "Set up a .env file with the API keys needed to access the models you want to use.\n[Set API Keys](command:vscode-aiconfig.setApiKeys)",
+            "description": "Define your model API keys in an ``.env`` file\n[Set API Keys](command:vscode-aiconfig.setApiKeys)",
```

Comment:
nit, an -> a

Reply:
I'm gonna keep as "an" because I feel most people would say "env" instead of "dot env" but yea I know what you mean, can fix later if we want